config/jobs/kubernetes/sig-network/ingress-gce-e2e.yaml

```diff
@@ -167,10 +167,10 @@ postsubmits:
         - --
         - --env=VERSION=$(PULL_BASE_REF)
         - --env=REGISTRY=gcr.io/k8s-ingress-image-push
-        - make
-        - all-push
-        - ALL_ARCH=amd64 arm64
-        - CONTAINER_BINARIES=e2e-test
+        - --env=ALL_ARCH="amd64 arm64"
+        - --env=ARCH=$(dpkg --print-architecture)
```

Comment:
I don't think this is correct, ARCH should define the target arch, not the one the job is running

Reply:
Do you know how to get a target ARCH in the script that test-infra runs?

Reply:
Ok, then maybe I misunderstood it, why do you need the current arch for crosscompiling?

Reply:
ok, got it, but the infra doesn't have arm workers at this moment so we can omit that step by now, besides is more correct as you say

we can also migrate the job, this will result in something like

```
  kubernetes/ingress-gce:
  - name: ci-ingress-gce-image-push
    decorate: true
    labels:
      preset-service-account: "true"
      preset-k8s-ssh: "true"
      preset-dind-enabled: "true"
    spec:
      containers:
      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230328-17e539c80f-master
        command:
        - runner.sh
        - hack/run-make-cmd.sh
        securityContext:
          privileged: true
        env:
        - name: ALL_ARCH
          value: "amd64 arm64"
        - name: BINARY
          value: "e2e-test"
    annotations:
      testgrid-dashboards: sig-network-ingress-gce-e2e
      testgrid-tab-name: ingress-gce-image-push
      testgrid-alert-email: kubernetes-sig-network-test-failures@googlegroups.com
```